src/Reflection/ReflectionFunctionAbstract.php

```diff
@@ -188,6 +190,21 @@ public function isVariadic(): bool
         return false;
     }
 
+    /**
+     * Check if the function throw error/exception.
+     */
+    public function isThrowable(): bool
```

Comment:
Rather than `isThrowable`, I'd say `containsThrowExpressions` or such: we only know if `throw` is used in there, but not if any callee will throw.

Also, `isThrowable` makes it look like this function is an instance of `Throwable`, which is quite confusing.

What about closures and anonymous classes produced in the body of a function? Should they affect the function declaring them? :thinking: